### ISSUE TITLE
[Mapping] Log Syntax Errors Immediately for Debugging

### ISSUE THREAD

**Issue by temanbrcom** [eclipse/che-che4z-lsp-for-cobol#342]
## Description ##
As a SE, I would like to see the syntax errors in the logs with the submitter specified so I can debug the server easier. 

## Acceptance Criteria ##
- Syntax errors logged with level "debug" when found
- Logging for all the errors at the end of analysis removed
- The log message contains the submitter class name. 

## Definition of Done ##
- The story is implemented according to the description.
- Unit testing done (Preferably by TDD approach, coverage should be at least 70%).
- Code review done by at least one other team member.
- Manual testing is done.
- Documentation is added and reviewed (if needed).
- The pull request is approved.
- The story is merged into the development branch.
- Tests are performed on the development branch.

## Technical details ##
This is the place for any implementation-related notes

**Comment by temanbrcom** [comment]
Blocked by #220
